server/views/pages/reportDetailMacro.njk

```diff
@@ -335,6 +335,17 @@
     }}
   {% endif %}
 
```

Comment:
taser question & answers will only show in reports (and check-your-answers) if the question was ever asked. This conditionality protects old reports that were never asked the question